src/apps/loan-list/materials/selectable-material/selectable-material.tsx

```diff
@@ -74,8 +75,7 @@ const SelectableMaterial: FC<SelectableMaterialProps & MaterialProps> = ({
           </div>
           <p className="text-header-h5 mt-8">{title}</p>
           <p className="text-small-caption">
-            {authors}
-            {year && <> ({year})</>}
+            <AuthorYear author={authors || ""} year={year || ""} />
```

Comment:
Great with a component for Author and year!
But I think it would be prettier if you make both props optional in the component.
Then you could just call it like:
```
<AuthorYear author={authors} year={year} />